brambling/views/core.py

```diff
@@ -115,7 +115,7 @@ def handle_invite(self):
         invite = self.invite
         content = self.content
         if invite.kind == Invite.EVENT:
-            content.create_order(self.request.user)
+            return Order.objects.for_request(event=content, request=self.request, create=True)
```

Comment:
This probably shouldn't return - just so that it matches the other sections of this func.